src/components/List/variantProduct.tsx

```diff
@@ -282,23 +280,20 @@ export const ExpendableContent: FunctionComponent<ExpandedContentProps> = ({
   children,
   id,
   rowsState,
-  isToggled,
-  ...props
 }) => (
-  <Box data-expandable {...props}>
+  <div data-expandable>
```

Comment:
This is also breaking

Reply:
I'm unsure about that one... console doesn't have any list.Expandable using box props and same inside SWUI it doesn't seem to require anything from Box

Reply:
You're right :)